packages/framework/src/Component/Image/ImageFacade.php

```diff
@@ -92,10 +93,36 @@ public function __construct(
         $this->mountManager = $mountManager;
     }
 
+    /**
+     * @param object $entity
+     * @param \Shopsys\FrameworkBundle\Component\FileUpload\ImageUploadData $imageUploadData
+     * @param string|null $type
+     */
+    public function manageImages(object $entity, ImageUploadData $imageUploadData, ?string $type = null): void
+    {
+        $imageEntityConfig = $this->imageConfig->getImageEntityConfig($entity);
+        $uploadedFiles = $imageUploadData->uploadedFiles;
+        $orderedImages = $imageUploadData->orderedImages;
+
+        if ($imageEntityConfig->isMultiple($type) === false) {
+            if (count($orderedImages) > 1) {
+                array_shift($orderedImages);
+                $this->deleteImages($entity, $orderedImages);
+            }
+            $this->uploadImage($entity, $uploadedFiles, $type);
+        } else {
+            $this->saveImageOrdering($orderedImages);
+            $this->uploadImages($entity, $uploadedFiles, $type);
+        }
+
+        $this->deleteImages($entity, $imageUploadData->imagesToDelete);
+    }
+
     /**
      * @param object $entity
      * @param array $temporaryFilenames
      * @param string|null $type
+     * @deprecated This function is deprecated since 7.3.2, use ImageFacade::manageImages instead
```

Comment:
about the message - Is it really deprecated or will it be changed to protected in the next release?

Reply:
Hi, thx. Fixed